test-defaults.ini

```diff
@@ -6,6 +6,9 @@ ws.auth_required = False
 
 is_test_running = True
 
+# Prevent plugins from loading during tests
+plugins_dir = '%(root)s/test_plugins'
```

Comment:
Minor suggestion to make that more self-documenting: use ``does_not_exist`` instead of ``test_plugins``.